core/src/jsMain/kotlin/com.saveourtool.processbuilder/utils/FileUtils.kt

```diff
@@ -0,0 +1,20 @@
+/**
+ * File containing utils for files
+ */
+
+package com.saveourtool.processbuilder.utils
+
+import com.saveourtool.processbuilder.NotImplementedInJsError
+import okio.FileSystem
+import okio.Path
+
+actual val fs: FileSystem = throw NotImplementedInJsError()
+
+/**
+ * Delete this directory and all other files and directories in it
+ *
+ * @param path a path to a directory
+ */
+actual fun FileSystem.myDeleteRecursively(path: Path) {
```

Comment:
please create some issue may be on this subject -- refactor/make sure that all existing methods are necessary

probably, some our methods could be replaced via latest okio methods
initially, we created them, because they were absent or incorrect in okio,
but a lot of time have left, may be they fixed these problems

Reply:
https://github.com/saveourtool/process-builder-multiplatform/issues/3

Reply:
also, we have the okio-extras for such extensions

Reply:
Will also create an issue in okio-extras to move it all the remaining methods there

Reply:
https://github.com/saveourtool/okio-extras/issues/36